develop/addons/schema-driven-forms/customising-form-behaviour/actions.rst

```diff
@@ -27,7 +27,7 @@ in our pizza order form:
         def handleCancel(self, action):
             ...
 
-The name of the method is not particularly important, so long as it is
+The name of the method is not particularly important, it only needs to be
```

Comment:
@svx thanks for the heads up. I agree with the goal to avoid these words in context where they could make something sound easy which not necessarily is easy for everybody. 
I will keep this in mind while working with the docs at the berlin-sprint. 
Even though I want to add, that the use in the sentence above is in my opinion a false positive (and I feel there are correct uses of the above words).

It was saying: *The name of the method is not particularly important, it only needs to be uniqe.* - I think, thats a different use than saying *You only have to import this 25 modules.*